README.md

```diff
@@ -96,3 +102,36 @@ remark()
 ```
 
 This allows you process files from one directory, and save the results to another.
+
+### Simple mode
+
+If the plugin is enabling graphs to be processed by [mermaidjs][mermaidjs],
```

Comment:
`The following code sample enables simple mode:` is probably all we need to say.

Otherwise, "as follows" instead of "like follows".

Reply:
Fixed